Packages/com.llealloo.audiolink/Runtime/Scripts/AudioLink.cs

```diff
@@ -124,10 +126,120 @@ public class AudioLink : MonoBehaviour
         private int _rightChannelTestDelay = 300;
         private int _rightChannelTestCounter;
         private bool _ignoreRightChannel = false;
+        
+        #region PropertyIDs
+
+        // ReSharper disable InconsistentNaming
+        
+        internal int _AudioTexture;
+        
+        // AudioLink 4 Band
+        internal int _FadeLength;
+        internal int _FadeExpFalloff;
+        internal int _Gain;
+        internal int _Bass;
+        internal int _Treble;
+        internal int _X0;
+        internal int _X1;
+        internal int _X2;
+        internal int _X3;
+        internal int _Threshold0;
+        internal int _Threshold1;
+        internal int _Threshold2;
+        internal int _Threshold3;
+        internal int _SourceVolume;
+        internal int _SourceDistance;
+        internal int _SourceSpatialBlend;
+        internal int _ThemeColorMode;
+        internal int _CustomThemeColor0;
+        internal int _CustomThemeColor1;
+        internal int _CustomThemeColor2;
+        internal int _CustomThemeColor3;
+        
+        // Global strings
+        internal int _StringLocalPlayer;
+        internal int _StringMasterPlayer;
+        internal int _StringCustom1;
+        internal int _StringCustom2;
+        
+        // Extra Properties
+        // internal int _EnableAutogain;
+        // internal int _AutogainDerate;
+        
+        // Set by Udon
+        internal int _AdvancedTimeProps0;
+        internal int _AdvancedTimeProps1;
+        internal int _PlayerCountAndData;
+        internal int _VersionNumberAndFPSProperty;
+
+        //Raw audio data.
+        internal int _Samples0L;
+        internal int _Samples1L;
+        internal int _Samples2L;
+        internal int _Samples3L;
+        
+        internal int _Samples0R;
+        internal int _Samples1R;
+        internal int _Samples2R;
+        internal int _Samples3R;
```

Comment:
any particular reason these are internal?

Reply:
they don't all have to be internal only like a handful but I wanted to be consistent and I don't see any downside